package.json

```diff
@@ -1,8 +1,8 @@
 {
-	"name": "vale-server",
+	"name": "vale",
```

Comment:
the renaming of the VS code extension name have some implications:
- users won't be automatically updated to this version
- it is possible to mark the old one as deprecated and mention that the migration should go to this new one. For that, we will need to mention it on this Github discussion https://github.com/microsoft/vscode-discussions/discussions/1

Reply:
in package-lock.json, teh name is still vale-server

Reply:
As we discussed when merging extensions, I don't think we should change this.

Reply:
@jdkato Personally I think it's really confusing now when there is no such thing as Vale server to call the extension vale server, but to avoid further confusion to users, let's keep it as is for now… Maybe I'll see if there's some way to handle it.